src/Twig/WebpackExtension.php

```diff
@@ -12,7 +12,7 @@
 
 use PMG\TwigWebpack\Webpack;
 
-final class WebpackExtension extends \Twig_Extension
+final class WebpackExtension extends \Twig\Extension\AbstractExtension
```

Comment:
Let's add a `use Twig\Extension\AbstractExtension` at the top here instead of a fully qualified class name here.